### ISSUE TITLE
Standard Employer Pitch Document

### ISSUE THREAD

**Issue by chrisdewar** [campjs/campjs-iii#8]
We should have some form of standard employer pitch document. Something simple that we can give to employers to convey the importance of this event, and the expected advantages and outcomes it will have for their dev staff.

My initial thoughts are that it should be a simple markdown file in a github repo, that we can then > html > pdf when required.

**Comment by iamkevinv** [comment]
+1

**Comment by shousper** [comment]
+1

**Comment by colingourlay** [comment]
+0.8

**Comment by niouton** [comment]
+3.14159265359

**Comment by timoxley** [comment]
Happy to do this, perhaps somebody could get a draft started in the [campjs wiki](https://github.com/campJS/campjs/wiki/_new) and I'll move it onto the site later

**Comment by timoxley** [comment]
i.e. writing copy is very slow and any help would be very appreciated.

**Comment by chrisdewar** [comment]
Yep, I'll have a look at this at some point today, if someone doesn't get to it first.

**Comment by timoxley** [comment]
Thanks @chrisdewar :kissing:

**Comment by iamkevinv** [comment]
I did some brief research and thinking on what it should contain:

Needs to include the basics of location, date, and cost (including approx travel and hotel, if needed, from different capitals around Australia), and can summarize what you'll learn.

It would be useful to explain exactly what we can get out of CampJS and how it's content benefits your employers and/or team. What is it's unique value proposition.
It should be able to demonstrate how the conference will contribute to some quantifiable result (x possible new networking connections, x areas of learning, x hours of engaging information, etc.
To go with this - a "take home pack" of information to disseminate to any team members that didn't get to go, or to be used for people who did visit... to have talking points for presenting to their teams in an infoshare session.
(Sending some people and getting them to come back and teach the group is a common practice!)

**Comment by timoxley** [comment]
@kevdesign great ideas

**Comment by rvagg** [comment]
I seem to remember JSConf.au having a good one of these; I don't suppose anyone knows where it might be located?

**Comment by timoxley** [comment]
@chrisdewar get a chance to draft this?

**Comment by MauriceButler** [comment]
I couldnt find one from JSConf but the one from WebDirections is here http://www.webdirections.org/downloads/wds13-why-send-the-team.pdf

**Comment by chrisdewar** [comment]
TBH I haven't had a lot of time to spend on this, but I have made a start on it. I will add it to the wiki now.

update: https://github.com/campJS/campjs/wiki/Employer-Pitch-Document
